tests/cupy_tests/math_tests/test_sumprod.py

```diff
@@ -131,6 +137,11 @@ def test_sum_keepdims(self, xp):
         a = testing.shaped_arange((2, 3, 4), xp)
         return a.sum(axis=1, keepdims=True)
 
+    @testing.numpy_cupy_allclose()
```

Comment:
```suggestion
    @testing.for_all_dtypes()
    @testing.numpy_cupy_allclose()
```

Reply:
done. Thanks

Reply:
I used the separate `src_dtype` and `dest_type` cases for one of these new tests